src/components/Notification/NotificationStack/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Notification } from 'reducers/notification';
+import ModalNote from './StackedNotification/Note/ModalNote';
+import StackedNotification from './StackedNotification';
+
+const getNotificationStyle = (index: number) => ({
+  top: `${2 + index * 8}rem`
+});
+const DEFAULT_DISMISS_TIME = 5000;
```

Comment:
Trengs denne? Er `NotificationDuration` required i typen? Ser for meg at det er et felt som kunne vært required. Om den fortsatt trengs, så ville jeg byttet den ut med en av `NotificationDuration`-enumene 😄

Reply:
Komponenten hade en optional prop som bestämde dismiss time, som defaultade till 5000. Tingen var bara att det aldrig sändes in något värde till den proppen, så då gjorde jag bara den till en konstant. 😅 Kan byta type!

Reply:
Jeg tror du egentlig kan endre i modellen, at dismiss er required? Så slipper du å sende inn en default

Reply:
Tog bort den som props, för den får aldrig in något värde och istället så har jag den bara som en konstant. Den ändrar på sig lite dock basert på vilket index det är längre ned i koden! 😊